packages/ses/src/tame-locale-methods.js

```diff
@@ -0,0 +1,56 @@
+import { assert } from './assert.js';
+import { getOwnPropertyNames, defineProperty } from './commons.js';
+
+const localePattern = /^(\w*[a-z])Locale([A-Z]\w*)$/;
+
+// See https://tc39.es/ecma262/#sec-string.prototype.localecompare
+const nonLocaleCompare = function localeCompare(that) {
```

Comment:
Fixing the omission of `localCompare` revealed yet another bug. Here, I define a replacement function with `function`. This does not adequately emulate a builtin method. `function` functions have constructor behavior and `.prototype` property. Builtin methods do not. Instead, we need to use concise method syntax.